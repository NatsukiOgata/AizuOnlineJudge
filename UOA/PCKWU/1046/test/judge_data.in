4 7
A#...#B
.#.#.#.
.#.#.#.
...#...
5
4 7
A#...#B
.#.#.#.
.#.#.#.
...#...
2
4 7
A#...#B
.#.#.#.
.#.#.#.
...#...
6
1 10
A#.......B
55555554
1 10
A#.......B
55555555
4 4
A...
....
....
..B.
62
4 4
A...
.#..
..#.
.#.B
4
4 4
A...
.#..
..#.
.#.B
46
4 4
A...
.#..
..#.
.#.B
462
4 4
A...
##.#
...#
.##B
82
4 4
A...
##.#
...#
.##B
4862
4 4
A...
##.#
..B#
.##.
4862
4 4
A...
##.#
...#
.##B
8
5 5
A.#..
..#..
.#...
...#.
###B.
4862
5 5
A.#..
..#..
.#...
...#.
###B.
8866
5 5
A.#..
..#..
.#...
...#.
###B.
4
6 6
......
.###..
..A#..
####.#
......
..B...
8
6 6
......
.###..
..A#..
####.#
......
..B...
88662244
6 6
......
.###..
..A#..
####.#
......
..B...
4488
7 7
A......
.......
.......
#######
.......
.B.....
.......
6644
10 10
A.........
.#.#...#..
.#.#...#..
.#...##...
.##.#...#.
....#.####
#.........
#.######.#
...B......
######.###
8866422
10 10
A.........
.#.#...#..
.#.#...#..
.#...##...
.##.#...#.
....#.####
#.........
#.######.#
...B......
######.###
886642
10 10
A.........
.#.#...#..
.#.#...#..
.#...##...
.##.#...#.
....#.####
#.........
#.######.#
...B......
######.###
48566226
10 10
A.........
.#.#...#..
.#.#...#..
.#...##...
.##.#...#.
....#.####
#.........
#.######.#
...B......
######.###
25888664
10 10
A.........
.#.#...#..
.#.#...#..
.#...##...
.##.#...#.
....#.####
#.........
#.######.#
...B......
######.###
25522644
10 10
A.........
.#.#...#..
.#.#...#..
.#...##...
.##.#...#.
....#.####
#.........
#.######.#
...B......
######.###
56584
10 10
A.........
.#.#...#..
.#.#...#..
.#...##...
.##.#...#.
....#.####
#.........
#.######.#
...B......
######.###
885264
10 10
A.........
.#.#...#..
.#.#...#..
.#...##...
.##.#...#.
....#.####
#.........
#.######.#
...B......
######.###
5
10 10
..........
.###......
.A.###....
.....###..
####..#...
..###.....
..B..##...
......#...
...###....
..........
888862
10 10
..........
.###......
.A.###....
.....###..
####..#...
..###.....
..B..##...
......#...
...###....
..........
226888
10 10
..........
.###......
.A.###....
.....###..
####..#...
..###.....
..B..##...
......#...
...###....
..........
8
10 10
..........
.###......
.A.###....
.....###..
####..#...
..###.....
..B..##...
......#...
...###....
..........
2
10 10
..........
.###......
.A.###....
.....###..
####..#...
..###.....
..B..##...
......#...
...###....
..........
66224488
15 15
...............
...............
......###......
........#A.....
.........####..
...............
...............
...............
.######........
.#...B#........
.#....#........
.######........
...............
...............
...............
88662244
15 15
...............
...............
......###......
........#B.....
.........####..
...............
...............
...............
.######........
.#...A#........
.#....#........
.######........
...............
...............
...............
88662244
1 6
A.B#..
6
6 6
######
#A..##
#..###
######
####B#
######
488622
16 16
................
................
.####...........
.#A.#...........
.####...........
................
...........##...
.........##.....
...........#....
.......####.....
#######.....###.
................
................
...B............
...###..........
................
5
16 16
................
A...............
.####...........
.#..#...........
.####...........
................
...........##...
.........##.....
...........#....
.......####.....
#######.....###.
................
................
...B............
...###..........
................
5
16 16
................
A...............
.####...........
.#..#...........
.####...........
................
...........##...
.........##.....
...........#....
.......####.....
#######.....###.
................
................
...B............
...###..........
................
88882222
16 16
................
A...............
.####...........
.#..#...........
.####...........
................
...........##...
.........##.....
...........#....
.......####.....
#######.....###.
................
................
...B............
...###..........
................
8668
20 20
....................
....................
......###...........
........#A..........
.........####.......
....................
....................
....................
.######.............
.#...B#.............
.#....#.............
.######.............
....................
....................
....................
....................
....................
....................
....................
....................
88662244
20 20
....................
....................
......###...........
........#A..........
.........####.......
....................
....................
....................
.######.............
.#....#.............
.#....#.............
.######.............
....................
....................
....................
.......###..........
........B.#.........
....................
....................
....................
88662244
20 20
......#.............
......#.............
......###...........
........#A..........
.........####......#
.............######.
....................
....................
.######.............
.#....#.............
.#....#.............
.######.............
....................
....................
....................
.......###..........
........B.#.........
....................
....................
....................
8866224546
20 20
......#.............
......#.............
........#...........
........#A..........
.........####......#
.............######.
....................
....................
.######.............
.#....#.............
.#....#.............
.######.............
....................
....................
....................
.......###..........
........B.#.........
....................
....................
....................
8866224546
20 20
......#.............
......#.............
........#...........
........#A..........
.........####......#
.............######.
....................
....................
.######.............
.#....#.............
.#....#.............
.######.............
..B.................
....................
....................
.......###..........
..........#.........
....................
....................
....................
8866224546
20 20
......#.............
......#.............
........#...........
........#A..........
.........####......#
.............######.
....................
....................
#######.............
.#....#.............
.#....#.............
.#######............
..B.....##..........
..........#.........
...........#........
.......###..####....
..........#.....#...
................#...
................#...
....................
8866224546
20 20
......#.............
......#.............
........#...........
........#A..........
.........####......#
.............######.
....................
....................
#######.............
.#....#.............
.#....#.............
.#######............
........##..........
........B.#.........
...........#........
.......###..####....
..........#.....#...
................#...
................#...
....................
8866224546
20 20
......#.............
......#.............
........#...........
........#A..........
.........####......#
.............######.
....................
....................
#######.............
.#....#.............
.#....#.............
.#######............
........##..........
..........#.........
...........#........
.......###..####....
..........#.....#...
..........#.....#...
.B........#.....#...
..........#.........
8866224546
20 20
......#.............
......#.............
........#...........
........#A..........
.........####......#
.............######.
....................
....................
#######.............
.#....#.............
.#....#.............
.#######............
........##..........
..........#.........
..........##........
.......###..####....
..........#.....#...
..........#.....#...
.B........#.....#...
..........#.........
8866224546
20 20
......#.............
......#.............
........#...........
........#A..........
.........####......#
.............######.
....................
....................
#######.............
.#....#.............
.#....#.............
.#######............
........##..........
..........#.........
..........##........
.......###..####....
..........#.....#...
..........#.....#...
.B........#.....#...
..........#.........
8866224546
20 20
......#.............
......#.............
........#...........
........#A..........
.........####......#
.............######.
....................
....................
#######.............
.#....#.............
.#....#.............
.#######............
........##..........
..........#.........
..........##........
.......###..####....
..........#.....#...
..........#.....#...
.B........#.....#...
..........#.........
8866224546
20 20
......#.............
......#.............
........#...........
........#A..........
.........####......#
.............######.
....................
....................
#######.............
.#....#.............
.#....#.............
.#######............
........##..........
..........#.........
..........##........
.......###..####....
..........#.....#...
..........#.....#...
B.........#########.
..........#....#....
8866224546
20 20
......#.............
......#.............
........#...........
........#A..........
.........####......#
.............######.
....................
....................
#######.............
.#....#.............
.#....#.............
.#######............
........##..........
..........#.........
..........##........
.......###..####....
..........#.....#...
..........#.....#...
B.........#########.
..........#....#...#
8866224546
0 0
