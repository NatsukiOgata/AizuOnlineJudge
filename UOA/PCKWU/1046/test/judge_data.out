18 0 6
15 3 6
18 0 6
72 0 0
impossible
6 3 3
3 3 0
6 3 3
6 3 3
impossible
6 2 2
3 1 2
3 0 3
6 3 2
10 0 4
impossible
3 2 2
12 3 4
7 2 0
impossible
12 6 5
12 6 5
15 9 6
11 6 4
impossible
10 6 3
8 5 3
11 8 3
3 3 2
12 4 4
3 3 2
18 9 2
22 8 2
18 7 5
impossible
impossible
impossible
impossible
35 13 3
12 9 3
16 5 11
18 7 5
19 14 9
impossible
21 15 10
18 12 4
34 10 7
21 12 10
51 17 6
84 16 11
84 16 11
84 16 11
154 16 17
154 16 17
